source/config-file/protect-connection-strings.txt

```diff
@@ -0,0 +1,86 @@
+.. _compass-protect-connection-strings:
+.. _hide-connection-string-credentials:
+
+==========================================
+Hide Credentials in Your Connection String 
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+When you paste a connection string into the |compass| connection form, by
+default, |compass-short| shows credentials in plaintext. To hide credentials in 
+the connection string, use the ``protectConnectionStrings`` option. 
+
+About This Task
+---------------
+
+When ``protectConnectionStrings`` is enabled, users cannot perform the following 
+actions: 
+
+- Edit the connection string in the |compass-short| connection form 
+
+- Copy the connection string in the |compass-short| interface 
+
+- See the credentials when :ref:`exporting a query <compass-export-query>`
+
+Procedure
+---------
+
+To hide your your connection string credentials, enable the 
+``protectConnectionStrings`` option. 
+
+You can set this option in either: 
```

Comment:
[nit]

Based on our quality discussion yesterday, specifically around use of "that", "this", etc, I'm inclined to say that we should maybe edit this to be:

You can set the ``protectConnectionStrings`` option in either:

Reply:
Updating!

Just for clarification, are we trying to avoid the use of "this" altogether? My understanding from yesterday's convo was that we want to avoid using "this" by itself (i.e. "You can set _this_ in either.") but it's alright to use it when it's qualifying a noun/object such as "option" in this example.

Reply:
I think it depends. In this instance, I feel like the extra clarification of the option name is helpful for skimming, especially since it's its own paragraph, so readers would need to jump back up to the previous paragraph to see what "this" option means.